camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/spi/dsl/DslYamlTest.java

```diff
@@ -66,6 +71,63 @@ public void testDslSelf() throws Exception {
     }
 
     @Test
+    public void testDslSingleTag() throws Exception {
+        final Entity app = createAndStartApplication(
+                "services:",
+                "- type: " + BasicApplication.class.getName(),
+                "  brooklyn.tags:",
+                "    - hi");
+        assertTrue(getTagsEventually(app).contains(BrooklynTags.newNotesTag("hi")));
+    }
+
+    @Test
+    public void testDslMultipleTags() throws Exception {
+        final Entity app = createAndStartApplication(
+                "services:",
+                "- type: " + BasicApplication.class.getName(),
+                "  brooklyn.tags:",
+                "  - tag1",
+                "  - tAg2");
```

Comment:
Include a test for a tag of a primitive type, e.g. `- 3`? (see earlier comment).